packages/babel-core/test/fixtures/async/plugin-post-chaining/plugin.js

```diff
@@ -0,0 +1,35 @@
+const wait = t => new Promise(r => setTimeout(r, t));
+
+function pluginC({ types: t }) {
+  return {
+    post() {
+      this.file.ast.program.body[0].value = 'pluginC'
+    },
+  };
+}
+
+function pluginB({ types: t }) {
+  return {
+    inherits: pluginC,
+    async post() {
+      await wait(50);
```

Comment:
Could you make pluginC async and keep pluginB sync? So we have `async-sync-async`, testing both combinations :)

Reply:
Done, test for `pre` uses sync-async-async and test for `post` uses async-sync-async.